Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+function CardContainer(props) {
+    const { banner, content} = props;
+
+    return (
+        <div className="card">
+            <CardBanner imageSrc={banner.imageSrc} />
+            <CardContent  title={content.title} body={content.body} link={content.link} />
+        </div>
+    );
+}
+
```

Comment:
- Aaron great job on the Social Card assignment. The design resemble provided design file 
- Your code is clean and you are properly splitting up the code into multiple components.
- Your git workflow is fluid and You did amazing job on this assignment 👍